app/models/membership.rb

```diff
@@ -67,6 +67,11 @@ def discussion_readers
       where('discussion_readers.user_id = ?', user_id)
   end
 
+  def experienced!(key, toggle = true)
```

Comment:
^^ (I could see us down the road interested in experiences for other things, like 'has a group made a decision together?', or 'has a network shared a proposal between groups?')